### PR TITLE
Node.js Certified Developer program

### PR DESCRIPTION
**Pull request by ZibbyKeaton**
We are planning to announce this news today at 9am ET. It covers what this new program entails, timing of the program and the parties involved in creating this program. The press release was shared and sent to analysts. ccing @HackyGoLucky

### CODE REVIEW

**Comment by evanlucas** [comment]
Can you add a `.md` file extension to the file so it renders as markdown?

**Comment by ZibbyKeaton** [comment]
@evanlucas hmmm, I thought I did that by using this tool: http://markitdown.medusis.com. Do I need to go in here again and try to do this again?

**Review by indutny — APPROVED** [review]
Other than .md extension, which is definitely required, the most important question that I have is: will CTC get this certificates? 😉

**Comment by gibfahn** [comment]
@ZibbyKeaton your filename is [node.js-certified-developer-program](https://github.com/ZibbyKeaton/new.nodejs.org/blob/patch-3/locale/en/blog/announcements/node.js-certified-developer-program), you should just be able to rename the file to `node.js-certified-developer-program.md`.

**Comment by evanlucas** [comment]
@ZibbyKeaton it needs the `.md` file extension added to the file. There are also so additional characters in the metadata at the top of the file. Would you like for me to fix it up for you?

File: locale/en/blog/announcements/nodejs-certified-developer-program.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
++---
```
Comment (evanlucas) [inline locale/en/blog/announcements/nodejs-certified-developer-program.md]: Can you remove the `+` at the beginning of this line (as well as the next 7 lines). It should be in the format like:

```
---
title: ...
date: ...
category: ...
slug: ...
layout: blog-post.hbs
---
```

**Comment by ZibbyKeaton** [comment]
@evanlucas @indutny alright, I think I did this. But if I didn't feel free to go in there and make changes. I'm still trying to figure this all out and know I've probably missed a lot of little items. One thing is for sure, I will not get this certificate 😀

**Review by evanlucas — CHANGES_REQUESTED** [review]

**Comment by hackygolucky** [comment]
Hah @indutny, if anyone in the CTC doesn't pass this certificate, we have done something very wrong. Let me know if y'all want to write an item or be an alpha tester :)

**Comment by evanlucas** [comment]
@ZibbyKeaton Thanks! I went ahead and fixed up the metadata in the file since you said yall were trying to get this out soon. So, LGTM. @hackygolucky I'd be happy to help with either :]

**Review by evanlucas — APPROVED** [review]

**Review by hackygolucky — APPROVED** [review]

**Comment by indutny** [comment]
I can probably alpha-test it once it is ready.

**Comment by ZibbyKeaton** [comment]
@evanlucas thank you for helping me here!

**Pull request merged**